src/widgets/input/ionic-input.template.html

```diff
@@ -8,3 +8,6 @@
         [id]="'control' + layoutNode?._id" [readonly]="options?.readonly ? 'readonly' : null" [type]="layoutNode?.type">
     </ion-input>
 </ion-item>
+<ion-item *ngIf="options?.showErrors && options?.errorMessage">
```

Comment:
The textarea error message was only using a `<div>` so wonder why you are using a `<ion-item>` this time

Reply:
It should be a ion-item, I'll fix the text area message